model/user_autocomplete.go

```diff
@@ -17,6 +17,22 @@ type UserAutocompleteInTeam struct {
 	InTeam []*User `json:"in_team"`
 }
 
+type UserAutocomplete struct {
+	InSystem     []*User `json:"in_system"`
```

Comment:
so the response should look like this: 
{
  "users": [{..},{..}]
}
and only when we have OutOfChannel users should look like: 
{
  "users": [{..},{..}],
  "out_of_channel": [{..},{..}]
}